packages/arb-token-bridge-ui/src/hooks/useArbTokenBridge.ts

```diff
@@ -348,12 +360,14 @@ export const useArbTokenBridge = (
 
   const approveTokenL2 = async ({
     erc20L1Address,
-    l2Signer
+    l2Signer,
+    walletAddress
```

Comment:
Why is this being newly passed to the function? Also we will need to change the function signature in `arbTokenBridge.types.ts` as well..

Reply:
updated